js/bybit.js

```diff
@@ -2531,7 +2402,7 @@ module.exports = class bybit extends Exchange {
         //
         const id = this.safeStringN (trade, [ 'execId', 'id', 'tradeId' ]);
         const marketId = this.safeString (trade, 'symbol');
-        market = this.safeMarket (marketId, market, undefined, 'contract');
+        market = this.safeMarket (marketId, market, undefined, market['type']);
```

Comment:
@sc0Vu we can't assume `market` is always defined here otherwise it break
![image](https://user-images.githubusercontent.com/43336371/220954054-4a8203a8-ea8d-447a-98f3-43e73957d42d.png)